source/dealer.js

```diff
@@ -38,6 +38,7 @@ class Dealer {
   }
 
   addPlayer(userID) {
+    // logic for wait for new hand??
```

Comment:
The logic for wait for new hand is already done because current hand works off of "round[]" instead of player. we only touch the players array at the beginning of the round to add everyone's hand